packages/angular_devkit/build_angular/src/builders/web-test-runner/schema.json

```diff
@@ -0,0 +1,291 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "title": "Web Test Runner Target",
+  "description": "Web Test Runner target options for Build Facade.",
+  "type": "object",
+  "properties": {
+    "main": {
```

Comment:
The options were copied from Karma's schema, but not all of them are implemented yet and the WTR builder will warn when an unsupported option is used. I'll add more support over time.

Reply:
Is the intention here to allow users to switch back and forth to karma/simplify manual transition?

Reply:
More or less. We'll want this to be as close to Karma as we can. It may not be as perfectly straightforward as "update `@angular-devkit/build-angular:karma` -> `@angular-devkit/build-angular:web-test-runner`. It may not be exactly 1:1 and we'll have to decide on divergences as they arise, but I think this is a good starting point at least.